### PR TITLE
Fetch global workspace memberships

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

In https://github.com/dust-tt/dust/pull/7194, the assumptions was made that all workspace members had an associated entry in the global workspace group. It's not the case yet, until we consolidate, this PR fetched the user ids from the memberships table.

### CODE REVIEW

**Review by Fraggle — APPROVED** [review]

**Comment by PopDaph** [comment]
🙏🏻

File: front/lib/resources/group_resource.ts [hunk 1]
```diff
@@ -371,14 +371,25 @@ export class GroupResource extends BaseResource<GroupModel> {
...
-        workspaceId: owner.id,
-        startAt: { [Op.lte]: new Date() },
-        [Op.or]: [{ endAt: null }, { endAt: { [Op.gt]: new Date() } }],
-      },
-    });
+
+    let memberships: GroupMembershipModel[] | MembershipResource[];
+
+    // The global group does not have a DB entry for each workspace member.
+    // TODO(GROUPS_INFRA): Remove this once we consolidate memberships with group memberships.
```
Comment (spolu) [inline front/lib/resources/group_resource.ts:378]: What's the rationale for creating these entries?

Comment (flvndvd) [reply front/lib/resources/group_resource.ts:378]: Creating which entries? The groups memberships? If yes, we discussed that we could phase out the regular memberships table in favor of the group_memberships. Obviously, this is not part of the current milestone. Today, no one has a membership on the global group, but this abstraction should not be spread in the front end and the server should still return the whole list.

Comment (spolu) [reply front/lib/resources/group_resource.ts:378]: ack

**Pull request merged**